front/tests/pages/stdcm-page-model.ts

```diff
@@ -260,7 +260,7 @@ class STDCMPage {
       this.destinationChField,
     ];
     for (const field of emptyFields) await expect(field).toHaveValue('');
-    await expect(this.codeCompoField).toHaveValue('__PLACEHOLDER__');
+    await expect(this.codeCompoField).toHaveValue('MA100');
```

Comment:
the function is named `verifyAllFieldsEmpty()`, if this field has a default value it can't be empty, you should rename the function, maybe add a comment to be more explicit

Reply:
Indeed ! I will change that